modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,24 @@
+== Inline formatting
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+List bullets and list numbers can now be inline-formatted.
+
+Inline-formatting, also known as spot-formatting, is the direct formatting of an object. It contrasts with formatting applied to an object because that object is a particular type of object.
+
+If an end-user selects the entire contents of a list item and applies inline formatting — such as a color change or a font-size change — this spot-formatting is also applied to the list item’s associated bullet or number.
+
+Only the list bullets or list numbers associated with the selection take on the inline-formatting applied to the selection.
+
+If the selection having inline formatting applied is one list item (that is, if the selection runs from one `<li>` tag to its associated `</li>` tag), the bullet or number associated with the selected list item takes on the inline-formatting applied to the selection.
+
+If the selection is the entire list, (that is, if the selection runs from the list’s opening `<ol>` or `<ul>` tag to the closing `</ol>` or `</ul>` tag, or if the selection runs from the list’s first `<li>` tag to the last `</li>` tag), every bullet or number takes on the inline-formatting applied to the selection.
+
+IMPORTANT: If, after applying inline-formatting, a partial selection of the now inline-formatted material is made and said inline-formatting is removed from the partial selection, the list bullet or list number formatting will also be removed.
+
+
+=== PowerPaste Premium plugin support
+
+Users of the xref:introduction-to-powerpaste.adoc[PowerPaste Premium plugin] should note, this plugin does not, currently, support inline-formatted bullets or list numbers.
+
+Inline-formatting, as applied to the bullets or list numbers, is lost when such material is copied and pasted using the PowerPaste plugin.
```

Comment:
Should these be the same paragraph. Since it's important to know that powerpaste doesn't block inline formats from being applied to list elements. The first sentence could be read like that but it's clarified in the second sentence.

Reply:
Added a NOTE admonition explicitly reminding folk that PowerPaste continues to support inline formatting of list elements.